src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
```

Comment:
I was thinking the toggle should still be the major way to set/unset a wfh event. I wanted to keep `/wfh clear` in the code in the event that somebody is uncertain if they have a wfh event on a given day and want to be certain that they can clear the wfh for that day. If `/wfh clear` causes confusion, it might be better to remove that feature than to remove the toggle.